tests/Unit/NumericalAlgorithms/Spectral/Test_Clenshaw.cpp

```diff
@@ -87,13 +85,11 @@ void test_sine_series(const std::vector<double>& coefficients,
   const DataType clenshaw_value =
       Spectral::evaluate_clenshaw(coefficients, alpha, beta, sin_0x, sin_x);
   CHECK_ITERABLE_APPROX(clenshaw_value, true_value);
-  // CHECK(cos_0x == 1.0);
 }
 
 }  // namespace
-SPECTRE_TEST_CASE("Unit.Numerical.Spectral.Clenshaw.CosineExact",
+SPECTRE_TEST_CASE("Unit.Numerical.Spectral.Clenshaw.Generic",
```

Comment:
[optional] Unless it is disambiguating from another test or conveys some meaning, you can leave off the `.Generic`.